io/include/pcl/io/vlp_grabber.h

```diff
@@ -80,13 +80,34 @@ namespace pcl
       virtual std::string
       getName () const;
 
+      /** \brief Allows one to customize the colors used for each of the lasers. */
+      void
+      setLaserColorRGB (const pcl::RGB& color,
+                        uint8_t laserNumber);
+
+    protected:
+      static const uint8_t VLP_MAX_NUM_LASERS = 16;
+      static const uint8_t VLP_DUAL_MODE = 0x39;
+
     private:
+      /** \brief Allows one to customize the colors used for each number of the lasers. */
+      void
+      setLaserColorRGB (const pcl::RGB& color,
+                        unsigned int laserNumber);
```

Comment:
* Doc parameters
* Why is this method defined in private scope?

Reply:
Sorry, It is mistake.